src/IO/ReadBufferFromFileDescriptor.h

```diff
@@ -39,6 +39,10 @@ class ReadBufferFromFileDescriptor : public ReadBufferFromFileBase
     {
     }
 
+    virtual ~ReadBufferFromFileDescriptor() override
```

Comment:
This change looks wrong.

Reply:
This change is no longer required. Rebased the branch with master.